.eslintrc.js

```diff
@@ -8,8 +8,8 @@ module.exports = {
     plugins: ['prettier'],
     rules: {
         'prettier/prettier': ['error'],
-        'radix': 0,
-        'eqeqeq': 0,
+        radix: 0,
+        eqeqeq: 0,
```

Comment:
Is this correct modification?

Reply:
@jimni1222 

Yes It's correct.

As you seen the keys in the object, they don't have any quote. 
but keys which includes operators( / , - , etc... ) should cover with quote.

![image](https://user-images.githubusercontent.com/16040418/70973225-99058a00-20e8-11ea-9cba-662458ac4902.png)

Reply:
Oh, okay :) Thank you for explaination